jsontests/src/in_memory.rs

```diff
@@ -21,6 +21,7 @@ pub struct InMemoryAccount {
 	pub code: Vec<u8>,
 	pub nonce: U256,
 	pub storage: BTreeMap<H256, H256>,
+	pub transient_storage: BTreeMap<H256, H256>,
```

Comment:
It's more optimize to have only one HashMap for the whole TransientStorage like revm do: https://github.com/bluealloy/revm/blob/main/crates/primitives/src/state.rs#L9

Reply:
I think `HashMap` has some issues in no_std environments and `BTreeMap` is usually recommended instead.

More info: https://stackoverflow.com/a/65470850

I know that `jsontests` folder is only used for running the tests, but `BTreeMap` was already used everywhere and I do not think this PR should change it.

Reply:
My comment is not about hasMApvs BtreeMap but one map vs N map for N accounts.

Con you use one `BTreeMap` map with key (H160, H256) instead?

Reply:
Ha I see, thank for the explanation